test/test_main.py

```diff
@@ -1402,12 +1190,10 @@ def test_crash_action(self, request: pytest.FixtureRequest, fake_script: FakeScr
         assert "'foo' not found in 'bar'" in stderr
 
 
-class TestMainWithDispatchAsScript(_TestMainWithDispatch):
+class TestMainWithDispatchAsScript(_TestMain):
```

Comment:
I'm a bit unclear if perhaps dispatch-is-a-script should be the default, as ops-based charms don't have a custom dispatch (they could, but not a single charm in our collection does that), and charmcraft generates dispatch which is a bash script...

But then... I see references to DISPATCH_PATH from year 2020 in charmcraft, and yet there's a PR from Aug 2024 that "adds dispatch if it doesn't exist". So is it a long-standing feature or a new feature?